bundles/net.tourbook/src/net/tourbook/database/TourDatabase.java

```diff
@@ -111,10 +111,11 @@ public class TourDatabase {
     * <li>/net.tourbook.export/format-templates/mt-1.0.vm</li>
     * <li>net.tourbook.device.mt.MT_StAXHandler</li>
     */
-   private static final int TOURBOOK_DB_VERSION = 47;
+   private static final int TOURBOOK_DB_VERSION = 48;
 
-//   private static final int TOURBOOK_DB_VERSION = 48; // 22.X ??
+//   private static final int TOURBOOK_DB_VERSION = 49; // 22.X ??
 
+//   private static final int TOURBOOK_DB_VERSION = 48; // 22.3.1 ??
```

Comment:
You should not write a version number `22.3.1 ??` when the version is not yet valid as it is irritating me, better would be `22.x ??`